.mk/development.mk

```diff
@@ -69,3 +69,35 @@ undeploy-infra: undeploy-loki undeploy-grafana uninstall
 
 .PHONY: undeploy-all
 undeploy-all: undeploy-infra undeploy-sample-cr undeploy-sample-workload
+
+.PHONY: generate-flp-configuration
+generate-flp-configuration: ## Generate custom metrics configuration
+	@echo -e "\n==> Generate metrics configuration"
+	sudo rm -rf /tmp/network_definitions || true
+	sudo rm -rf /tmp/destDir || true
+	cp -r network_definitions /tmp/network_definitions
+	$(OCI_BIN) run --entrypoint /app/confgenerator -v /tmp/network_definitions:/app/network_definitions -v /tmp/destDir:/app/destDir quay.io/netobserv/flowlogs-pipeline:latest --srcFolder /app/network_definitions --destConfFile /app/destDir/flowlogs-pipeline.conf.yaml
+	# TODO: add --destDocFile and --destGrafanaJsonnetFolder
```

Comment:
can you move "quay.io/netobserv/flowlogs-pipeline:latest" into a makefile variable

Reply:
the tag `:latest` needs to be aligned with the version of FLP that is used by the operator (we do not want to use latest)

Reply:
changed to FLP_VERSION